client/src/components/VisualiseData.vue

```diff
@@ -99,15 +99,15 @@ export default {
     },
     mergeLinks(indexA, indexB) {
       for (let i = 0; i < this.changedData.links.length; i++) {
-        let dupa = 0;
```

Comment:
why was it even merged with this dupa XD

Reply:
I had no good idea on how to name this variable, so I thought ill change it later. And then forgot about it completely xd

Reply:
Well I understand you, Im just curious why anyone in review havent noticed that